core/src/main/resources/hudson/PluginManager/installed.properties

```
@@ -24,6 +24,7 @@ requires.restart=This Jenkins instance requires a restart. Changing the state of
 detached-disable=This plugin may not be safe to disable
 detached-uninstall=This plugin may not be safe to uninstall
 detached-possible-dependents=Its functionality was at one point moved out of Jenkins core, and another plugin with a core dependency predating the split may be relying on it implicitly.
+detached-many-dependents=There are still {0} plugins with an implied dependency installed.
```

```suggestion
detached-many-dependents=There are {0} plugins with an implied dependency installed.
```

"Still" indicates cleanup needs to be done when that's out of the hands of normal admins.

---

Good catch, thanks.